### PR TITLE
feat(CLI): Support envd images prune #976

### PR DESCRIPTION
**Pull request by enjoyliu**

### CODE REVIEW

File: pkg/app/image_prune.go [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+var CommandPruneImages = &cli.Command{
+	Name:   "prune",
+	Usage:  "Remove unused images",
+	Action: pruneImages,
+}
+
+func pruneImages(clicontext *cli.Context) error {
+
+	envdEngine, err := envd.New(clicontext.Context, "docker")
```
Comment (gaocegege) [inline pkg/app/image_prune.go:24]: We refactored the interface, maybe you can use github.com/tensorchord/envd/pkg/docker directly.

**Review by gaocegege — COMMENTED** [review]
Thanks for your contribution! :tada: :+1:

File: pkg/app/image_prune.go (hunk 1 above)
Comment (enjoyliu) [reply pkg/app/image_prune.go]: ok i just found out. I've already changed it.

Comment (enjoyliu) [reply pkg/app/image_prune.go]: There are some other issues here, I found that docker's prune has some args, do we need to support？

File: README.md [hunk 2]
```diff
@@ -319,6 +319,9 @@ Thanks goes to these wonderful people ([emoji key](https://allcontributors.org/d
     <tr>
       <td align="center"><a href="https://www.homeboyc.cn/"><img src="https://avatars.githubusercontent.com/u/22193008?v=4?s=70" width="70px;" alt="杨成锴"/><br /><sub><b>杨成锴</b></sub></a><br /><a href="https://github.com/tensorchord/envd/commits?author=asjdf" title="Code">💻</a></td>
     </tr>
+    <tr>
+      <td align="center"><a href="https://www.homeboyc.cn/"><img src="https://avatars.githubusercontent.com/u/22193008?v=4?s=70" width="70px;" alt="杨成锴"/><br /><sub><b>杨成锴</b></sub></a><br /><a href="https://github.com/tensorchord/envd/commits?author=asjdf" title="Code">💻</a></td>
+    </tr>
```
Comment (gaocegege) [inline README.md:324]: Please do not update the section manually, I will use the bot to automatically generate it.

**Comment by gaocegege** [comment]
Generally LGTM!

Thanks for your contribution! :tada: :+1:

File: README.md (hunk 2 above)
Comment (enjoyliu) [reply README.md]: I makd some mistakes when rebase, now it was delelted.

**Comment by gaocegege** [comment]
The CI failure is not caused by your PR. The openvsx.org is down. 

I will merge the PR manually if e2e-lang passes.

Thanks for your contribution! :tada: :+1:

**Comment by gaocegege** [comment]
@all-contributors Please add @enjoyliu for the code contribution

**Pull request merged**